cos/static/css/prism.css

```diff
@@ -5,6 +5,13 @@
  * @author Lea Verou
  */
 
+ body {
+	 min-width: 310px;
+	 width: auto !important;
```

Comment:
As a general rule, !important tags are an admission of defeat. It would be better to figure out what the underlying problem is and fixing it properly, because sooner or later it's going to cause problems. I'm okay with either doing that now or making a new medium-priority ticket to clean it up.